src/test/features/claim/routes/check-and-send.ts

```diff
@@ -85,10 +85,10 @@ describe('Claim issue: check and send page', () => {
           .get(ClaimPaths.checkAndSendPage.uri)
           .set('Cookie', `${cookieName}=ABC`)
           .expect(res => expect(res).to.be.successful.withText('Check your answers'))
-          .expect(res => expect(res).to.be.successful.withText('<a href="/claim/claimant-individual-details" class="bold">Change <span class="visuallyhidden">full name</span></a>'))
```

Comment:
Rather than changing in so many places why not have it in a variable and update test cases accordingly.